samza-core/src/main/java/org/apache/samza/coordinator/leaderelection/LeaderElector.java

```diff
@@ -20,6 +20,8 @@
 package org.apache.samza.coordinator.leaderelection;
 
 import org.apache.samza.annotation.InterfaceStability;
+import org.apache.samza.coordinator.LeaderElectorListener;
+
 
 @InterfaceStability.Evolving
 public interface LeaderElector {
```

Comment:
I don't think there is a need for cancel() in our implementation. Everyone just calls tryToBecomeALeader(), and only one will get a callback. Do you think we should have an api like this, in case, there is another implementation?